pkg/discovery/dtofactory/application_entity_dto_builder.go

```diff
@@ -18,7 +17,8 @@ import (
 )
 
 const (
-	AppPrefix string = "App-"
+	AppPrefix                  string  = "App-"
+	defaultTransactionCapacity float64 = 50.0
```

Comment:
let's make the default Transaction Capacity 500. 50 is more like a number we used for testing (to get provision action easier). In the future it should be configurable from the policy tab by user. It's a app by app scenario.